ONSdigital/eq-author-app
# Validate right side of routing and skip

### What is the context of this PR?

Adds warning error and message to number, checkbox and radio answers when the right hand side is empty

### How to review 

1. Create questionnaire
2. Add three questions of types number, checkbox and radio
3. Go to each logic tab for each and add routing and skip condition using each answer type
4. Red border should surround the answer picker with either Enter a number (number) or Select at least one option
(radio, checkbox)

Codecov is back
Codacy is lame
